assets/css/style.css

```diff
@@ -889,6 +889,12 @@ tbody {
 
 /* side list */
 
+.sidebar-scroll {
+    overflow: auto;
+    height: 5rem !important;
+    padding-left: 0.5rem;
```

Comment:
Instead of using `rem` (which is tied to the font size of the root element), what happens if we try using `vh` (which is tied to the height of the viewport, or visible screen)? I suspect this will be more responsive for us. Bigger displays will show a longer navigation, and smaller displays will be more condensed.

Could you experiment with `vh` and see if you can make something that expands to size on larger displays?

More details on [values and units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units).

Reply:
Resolved in commit cb0dea0bd2fc2407519a1c65ae808af904104535.